app/controllers/submit.js

```diff
@@ -357,8 +369,16 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
         return changed;
     }),
 
+    basicsOriginalPublicationDate: Ember.computed.or('model.originalPublicationDate'),
```

Comment:
I'm not sure why you're using the computed or macro here. Maybe I'm just not following the logic?

Reply:
After our discussion last week, I've tried to initialize the `basicsOriginalPublicationDate` and `basicsDOI` within the `clearFields()`, which would be invoked upon controller setup. However, turns out that `clearFields()` would only be called if `model.isLoaded` is set, therefore would not successfully initialize. I decided to keep it this way as it is kind of a pattern and also it works. Tell me what you think.

Reply:
I'm ok with this as long as you add a comment above it explaining what's going on. I think a computed alias would also be clearer than a computed or.